### DISCUSSION TITLE
Cask Minecraft Java Version Question

### DISCUSSION THREAD

**Discussion by andrewcrook** [Homebrew/discussions#3466] [Casks]
What version is used for the Minecraft cask ?

Currently the brew casks reports  `1.1.26`

I have reinstalled from brew-casks and also checked autoupdates.  
However the latest `Minecraft Launcher` is `version 1.1.21 (1)`
Current `Minecraft for Java` is currently `1.19.1`

`Brew livecheck` reports latest is 
`minecraft: 1.1.26 ==> 1.1.26 `

However, this doest match the version installed.

**Comment by SMillerDev** [comment]
Brew extracts the version from the file at: https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/minecraft.rb#L5 and that says the version is 1.1.26.

**Reply by andrewcrook** [reply]
it must be the version of the installer

**Reply by SMillerDev** [reply]
No, it's the version encoded in the app file.

**Reply by aaronliu0130** [reply]
Additionally, Minecraft for mac doesn't have an installer.
